### ISSUE TITLE
Accessibility: Focus visible on tabs

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#209]
Me again :) Sorry for all of the issues - I have been logging them as I come across anything I see just to make sure I don't forget!

On pages that have tabs (like the [Payroll](https://www.jccmi.edu/business-office/payroll/) page), the [SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc?hl=en-US) checker is coming up with an issue for each active tab: 

**Highlighting Elements:**
With keyboard navigation it is important that the focused component can be visually identified. If the focused component is not somehow highlighted, the user cannot be sure which component has focus.

Make sure that user interface elements, that can receive keyboard focus, are highlighted on focus.

**Comment by joelworsham** [comment]
No apologies! Opening these issues is a wonderful thing. We constantly want to be improving the site. I would never complain about an abundance of improvements logged for the site.

This could be another candidate for upgrading Foundation framework to fix on its own, but I'll check.

**Comment by joelworsham** [comment]
Could you do me a favor and re-check this? I'm wondering if the update to Foundation 6 may have fixed this. Thanks!

**Comment by banksashleym** [comment]
Hi Joel,
Unfortunately not, this issue is still coming up on pages that incorporate tabs. Thanks!

**Comment by joelworsham** [comment]
Shucks. No problem! I'll hit this today.

**Comment by joelworsham** [comment]
Fixed in f3244536

**Comment by banksashleym** [comment]
Hi Joel, thanks! For some reason this one is still showing up...I tested a page with these tabs incorporated and pressed "tab" to navigate through the page. It jumped and highlighted all the links but it didn't ever jump to the links in these tabs, if that makes sense? I think it's saying that they're missing keyboard focus or something? Thanks!

**Comment by joelworsham** [comment]
Ahhh I see. I'll make sure this is working next week.

**Comment by joelworsham** [comment]
Think I found it! I've got the tabs now outlined properly on focus. Here's me testing it out with only they  keyboard:

![](https://d.pr/i/nzByC1+)

Let me know if this comes up solved! (check after this issue is officially closed out)
